### PR TITLE
Add helm repo management functions and fix test post stable chart deprecation

### PR DESCRIPTION
**Pull request by yorinasub17**
See https://helm.sh/blog/new-location-stable-incubator-charts/ for more details

### CODE REVIEW

File: modules/helm/install_test.go [hunk 1]
```diff
@@ -48,6 +46,12 @@ func TestRemoteChartInstall(t *testing.T) {
 		},
 	}
 
+	// Add the stable repo under a random name so as not to touch existing repo configs
+	uniqueName := random.UniqueId()
```
Comment (brikis98) [inline modules/helm/install_test.go:50]: NIT: perhaps add some sort of prefix to this random name (e.g., `terratest-`) so it's a bit easier to know where it came from?

**Review by brikis98 — APPROVED** [review]
LGTM!

File: modules/helm/install_test.go (hunk 1 above)
Comment (yorinasub17) [reply modules/helm/install_test.go]: Updated!

**Comment by yorinasub17** [comment]
Addressed nits! Can I have another review please?

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by yorinasub17** [comment]
Thanks for review! Helm tests are passing now so will merge this in.

**Pull request merged**